aspnetcore/mvc/views/tag-helpers/built-in/samples/TagHelpersBuiltIn/Pages/Products.cshtml.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.AspNetCore.Mvc.RazorPages;
+using System.Collections.Generic;
+using TagHelpersBuiltIn.Models;
+
+namespace TagHelpersBuiltIn.Pages
+{
+    public class ProductsModel : PageModel
+    {
+        public List<Product> Products { get; set; }
```

Comment:
I've been leaning toward private setters on these. By "leaning," I mean I'm not really 100% sure that I've been 100% consistent about it, but I have been making an effort.

Reply:
That makes a lot of sense in this case, actually.